lessons/Data-Science/ai-linear-regression/langs/en-us/pages/alvinn-training.adoc

```diff
@@ -0,0 +1,61 @@
+= Supervised Machine Learning: Training ALVINN
+
+@n Imagine that ALVINN has trained on the 15 images collected during a half minute of driving. How do you predict its best guesses about the steering angle will be? Explain. @fitb{}{}
+
+@fitb{}{@ifsoln{Given that 15 images is a relatively small amount, ALVINN's steering angle guesses will be essentially random.}}
+
+@fitb{}{@ifsoln{A function built on just 15 ordered pairs is not going to have a high level of accuracy.}}
+
+@fitb{}{}
+
+
+@n Respond to the questions below, considering how each variation in training will influence ALVINN's performance.
+
+@vspace{1ex}
+
+@indented{a) ALVINN has trained on the images collected during several _hours_ of driving on *an isolated one-lane road* on a sunny day. How do you think ALVINN's best guesses about the steering angle will be on the designated road? Explain. @fitb{}{}
+
+@fitb{}{@ifsoln{ALVINN's best guesses about the steering angle will be accurate - but only for that particular road on a sunny day.}}
+
+@fitb{}{}
+
+@fitb{}{}
+
+
+b) Predict how safely ALVINN will drive on that same road on a sunny, snowy day. Explain. @fitb{}{}
```

Comment:
I think I'm getting hung up on "predict how safely", because it makes me think you want like a percentage or something. How about "Do you think ALVINN will drive just as well on that same road on a sunny, snowy day? Why or why not?"